pandas/core/generic.py

```diff
@@ -9904,8 +9906,9 @@ def _check_percentile(self, q):
         2   -0.055556
         dtype: float64
 
-        See the percentage change in a Series where filling NAs with last
-        valid observation forward to next valid.
+        See the computing of percentage change in a Series with NAs. With
+        default skipped NAs, NAs are ignored before the computation and kept
```

Comment:
can you reword this a bit. Its not clear what this is doing. This can be in the Notes section.